docs/fundamentals/code-analysis/style-rules/using-directive-preferences.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "using directive preferences"
+description: "Learn about code analysis language rules for 'using' directive preferences"
+ms.date: 09/30/2020
+ms.topic: reference
+author: gewarren
+ms.author: gewarren
+dev_langs:
+- CSharp
+- VB
+---
+# 'using' directive preferences
```

Comment:
For the language rules, it looks like we don't have an overview page when there's only a single rule in the section. I think that makes more sense that having this sort of "extra" file that doesn't add much value.

Reply:
Sure, let me delete this. We can introduce it if we end up adding another rule to this option group.